### PR TITLE
fix: remove some tabText() in groupWidget when rename group

### PR DESCRIPTION
**Pull request by xiaoyifang**
relate #1484

### CODE REVIEW

**Comment by xiaoyifang** [comment]
@shenlebantongying  can you help to check on KDE environment when available?

**Comment by shenlebantongying** [comment]
The https://github.com/xiaoyifang/goldendict-ng/issues/1484 issue for edit groups is gone after this change in KDE.

**Comment by shenlebantongying** [comment]
The group name part of https://github.com/xiaoyifang/goldendict-ng/issues/1484 is fixed for KDE after this change.

File: src/ui/groups_widgets.cc [hunk 1]
```diff
@@ -910,13 +910,13 @@ void DictGroupsWidget::groupsByMetadata()
...
 }
 
-
 QString DictGroupsWidget::getCurrentGroupName() const
 {
   const int current = currentIndex();
 
   if ( current >= 0 ) {
-    return Utils::unescapeAmps( tabText( current ) );
+    auto * w = dynamic_cast< DictGroupWidget * >( widget( current ) );
```
Comment (shenlebantongying) [inline src/ui/groups_widgets.cc:918]:
```suggestion
    auto * w = qobject_cast< DictGroupWidget * >( widget( current ) );
```

**Review by shenlebantongying — APPROVED** [review]
Is there something more to do about this PR?

**Comment by xiaoyifang** [comment]
Consider it done at this time.

**Pull request merged**